<launch>
  <node name="sbg_ellipse" pkg="sbg_driver" type="ellipse" output="screen">
  	<param name="uart_port" value="/dev/ttyUSB0" />
  	<param name="uart_baud_rate" value="115200" type="int" />

	<remap from="imu" to="raw_imu" />
  </node>
</launch>